### PR TITLE
PR: Add support to PyQt4-sip 4.19.13

### PR DESCRIPTION
**Pull request by milanmatic**
Hello maintainers,

I am using PyQt4-sip==4.19.13 and I find qtpy very useful.
I would like to add support for my version of PyQt without impacting other versions.

Please review PR and let me know if I should change anything. 

I would like to have this PR official in pip ASAP, so let me know if there is anything else I could help with.

Thank you in advance.
Best regards,
Milan

Thank you

### CODE REVIEW

File: qtpy/QtGui.py [hunk 1]
```diff
@@ -33,7 +33,10 @@
                                  qFuzzyCompare)
     except ImportError:
         pass
-    from PyQt4.Qt import QKeySequence, QTextCursor
+    try:
```
Comment (goanpeca) [inline qtpy/QtGui.py]: Could you add some comments here on why this is needed like:

```
# PyQt 4.x.x requires .....
```

File: qtpy/__init__.py [hunk 2]
```diff
@@ -188,8 +188,12 @@ class PythonQtWarning(Warning):
...
             # PyQt < v4.6
             pass
-        from PyQt4.Qt import PYQT_VERSION_STR as PYQT_VERSION  # analysis:ignore
-        from PyQt4.Qt import QT_VERSION_STR as QT_VERSION  # analysis:ignore
+        try:
+            from PyQt4.Qt import PYQT_VERSION_STR as PYQT_VERSION  # analysis:ignore
+            from PyQt4.Qt import QT_VERSION_STR as QT_VERSION  # analysis:ignore
+        except ImportError:
+            from PyQt4.QtCore import PYQT_VERSION_STR as PYQT_VERSION  # analysis:ignore
+            from PyQt4.QtCore import QT_VERSION_STR as QT_VERSION  # analysis:ignore
```
Comment (goanpeca) [inline qtpy/__init__.py]: Also here

```
# PyQt 4.x.x requires .....
```

**Comment by goanpeca** [comment]
Thanks for the PR, I believe we are fading out support for 4.x but this PR is welcome :-)

File: qtpy/QtGui.py (hunk 1 above)
Comment (milanmatic) [reply qtpy/QtGui.py]: Sure, I will

Comment (goanpeca) [reply qtpy/QtGui.py]: Thanks :-)

**Comment by milanmatic** [comment]
@goanpeca thanks for quick response.

**Comment by milanmatic** [comment]
Hello,

Any news when it could be merged and published under pip? 
Do you need any action on my side?

Thank you.

**Comment by goanpeca** [comment]
@ccordoba12 any final thoughts here?

File: qtpy/QtGui.py [hunk 3]
```diff
@@ -33,7 +33,11 @@
...
+        from PyQt4.Qt import QKeySequence, QTextCursor
+    except ImportError:
+        # In PyQt4-sip==4.19.13 QKeySequence and QTextCursor are in PyQt4.QtGui
```
Comment (ccordoba12) [inline qtpy/QtGui.py]: Please change this to

    In PyQt4-sip 4.19.13 ...

File: qtpy/__init__.py [hunk 4]
```diff
@@ -188,8 +188,13 @@ class PythonQtWarning(Warning):
         except (AttributeError, ValueError):
             # PyQt < v4.6
             pass
-        from PyQt4.Qt import PYQT_VERSION_STR as PYQT_VERSION  # analysis:ignore
-        from PyQt4.Qt import QT_VERSION_STR as QT_VERSION  # analysis:ignore
+        try:
+            from PyQt4.Qt import PYQT_VERSION_STR as PYQT_VERSION  # analysis:ignore
+            from PyQt4.Qt import QT_VERSION_STR as QT_VERSION  # analysis:ignore
+        except ImportError:
+            # In PyQt4-sip==4.19.13 PYQT_VERSION_STR and QT_VERSION_STR are in PyQt4.QtCore
```
Comment (ccordoba12) [inline qtpy/__init__.py]: Please change this to

    # In PyQt4-sip 4.19.13 ...

**Comment by milanmatic** [comment]
@ccordoba12 I have updated the comment in code explaining why the PR is needed.
I have also fixed typo in previous commit message and pushed again.

**Comment by goanpeca** [comment]
Thanks for working on this!

**Pull request merged**